x-pack/plugins/ml/public/application/model_management/force_stop_dialog.tsx

```diff
@@ -160,6 +182,30 @@ export const StopModelDeploymentsConfirmDialog: FC<ForceStopModelConfirmDialogPr
           </EuiCallOut>
         </>
       ) : null}
+
+      {inferenceServiceIDs.length > 0 ? (
+        <>
+          <EuiCallOut
+            title={
+              <FormattedMessage
+                id="xpack.ml.trainedModels.modelsList.forceStopDialog.inferenceServicesWarning"
+                defaultMessage="The following {inferenceServicesCount, plural, one {deployment is} other {deployments are}} used by _inference API and can not be stopped:"
+                values={{ inferenceServicesCount: inferenceServiceIDs.length }}
```

Comment:
We'll need a message which conveys that there are some inference services, but they don't have permission to see them (although we shouldn't explicitly say that).
`hasInferenceServices` isn't yet being used in the UI.

Reply:
Added in https://github.com/elastic/kibana/pull/175880/commits/f9bb803f55dc07e093cb17ece6d4153d9c7926ec